bot/exts/moderation/infraction/management.py

```diff
@@ -232,6 +236,9 @@ async def infraction_edit(
             # If the infraction was not marked as permanent, schedule a new expiration task
             if request_data["expires_at"]:
                 self.infractions_cog.schedule_expiration(new_infraction)
+                # Timeouts are handled by Discord itself, so we need to edit the expiry in Discord as well
+                if user and infraction["type"] == "timeout":
+                    await user.edit(reason=reason, timed_out_until=expiry)
```

Comment:
We need to cap expiry at 28 days since that is the maximum timeout duration supported by Discord. The moderator should also be notified of this.

Reply:
5abcfaf564897b10234799d924bf1e8ca7fc7034